lib/gui/models/drives.js

```diff
@@ -20,103 +20,94 @@
  * @module Etcher.Models.Drives
```

Comment:
@Shou Either that, or have a localStorage shim for nodejs (like `node-localstorage`). Feel free to experiment with it on another PR if you like.